package.json

```diff
@@ -11,9 +11,10 @@
   },
   "scripts": {
     "start": "node dist/server.js",
+    "debug": "node --inspect dist/server.js",
     "build": "rimraf dist && tsc -p tsconfig.prod.json",
     "release": "npm run build && ./scripts/release.sh",
-    "playwright": "npm run build && docker-compose -f docker-compose.playwright.yml up --build -d && cd playwright && playwright test && docker-compose -f ../docker-compose.playwright.yml down",
+    "playwright": "npm run build && docker compose -f docker-compose.playwright.yml up --build -d && cd playwright && playwright test && docker compose -f ../docker-compose.playwright.yml down",
```

Comment:
Is this a recent change to `docker`? I've always used `docker-compose`

Reply:
@aflesher https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose